lib/gollum/public/gollum/stylesheets/template.scss.erb

```diff
@@ -2,72 +2,6 @@
 
 @import "_base", "_breakpoint", "_features";
 
-/*!
- *  Font Awesome 4.0.3 by @davegandy - http://fontawesome.io - @fontawesome
- *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
- */
-@font-face {
-  font-family: 'FontAwesome';
-  src: asset-url('fontawesome-webfont.eot?v=4.0.3');
-  src: asset-url('fontawesome-webfont.eot?#iefix&v=4.0.3') format('embedded-opentype'), asset-url('fontawesome-webfont.woff?v=4.0.3') format('woff'), asset-url('fontawesome-webfont.ttf?v=4.0.3') format('truetype'), asset-url('fontawesome-webfont.svg?v=4.0.3#fontawesomeregular') format('svg');
-  font-weight: normal;
-  font-style: normal;
-}
-
-.fa {
-  display: inline-block;
-  font: normal normal 16px FontAwesome;
-  line-height: 1;
-  text-decoration: none;
-  @include font-smoothing(on);
-}
-
-.fa-link:before {
-  content: "\f0c1";
-}
-
-.fa-spinner:before {
```

Comment:
Actually, don't we use the spinner for uploading?

Reply:
Good point! That probably broke after I removed the fonts anyway. Maybe there is a Primer replacement?

I'll search the source for ` fa ` and `fa-` to see if we forgot about more features.